operations/README.md

```diff
@@ -20,6 +20,11 @@ Role Variables
     # Docker version to update to.
     osbs_docker_version: <default not set>
 
+    # Update OpenShift node labels.
+    osbs_update_node_labels: false
+    # A list of labels to be applied to each OpenShift node.
+    osbs_node_labels: []
+
```

Comment:
README should also mention `osbs_managed_node_labels`

Reply:
ok